.gitignore

```diff
@@ -1,5 +1,6 @@
 .env
 node_modules/
+build/out
```

Comment:
This might need some investigation, since the gitignore in the agent folder already seem to have this.

Reply:
ye, it's not needed, deleted